packages/adena-extension/src/hooks/use-add-account.tsx

```diff
@@ -1,6 +1,7 @@
 import { useRecoilState, useResetRecoilState } from 'recoil';
 import { WalletState } from '@states/index';
 import { useAdenaContext } from './use-context';
+import { SeedAccount } from 'adena-module/src';
```

Comment:
@jinoosss Is src really necessary?

Reply:
I think I need to fix this.
I think we need to fix the type build part of the adena-module.